pkg/vm/memory/memory_value.go

```diff
@@ -331,3 +331,21 @@ func (memory *Memory) GetConsecutiveMemoryValues(addr MemoryAddress, size int16)
 	}
 	return values, nil
 }
+
+func ResolveAsBigInt3(memory *Memory, valAddr MemoryAddress) ([]*f.Element, error) {
```

Comment:
i think we should return a `[3]*fp.Element` instead of a `[]*fp.Element`

Reply:
Hi @TAdev0 Changes made!